salt/scripts.py

```diff
@@ -46,9 +46,24 @@ def _handle_signals(client, signum, sigframe):
         hardcrash = client.options.hard_crash
     except (AttributeError, KeyError):
         hardcrash = False
+
+    exit_msg = '\nExiting gracefully on Ctrl-c'
```

Comment:
After some more testing, I pushed https://github.com/saltstack/salt/pull/36750/commits/0e7c600e02509654a0a4cc89d937e5476e4cb60f which only includes the "Ctrl-c" message if the handler is triggered because of a SIGINT.

However, I noticed something weird about how the handlers are registered in scripts.py. See [here](https://github.com/saltstack/salt/blob/8ff69bf0c7b2e83126428de187eb5f99dbc444a0/salt/scripts.py#L61-L63) We're registering SIGINT instead of SIGTERM here. If I change this from SIGINT to SIGTERM (line 63 in that view), then it takes two times hitting Ctrl-c to stop the the salt CLI command for some reason I don't yet grok.

Also, since we're catching ``SystemExit`` [here](https://github.com/saltstack/salt/blob/8ff69bf0c7b2e83126428de187eb5f99dbc444a0/salt/client/mixins.py#L359), the ``SystemExit`` raised by the handler ends up formatted with full traceback in the runner output.

Thoughts?